### DISCUSSION TITLE
Allowing reactors that are touching to share fuel

### DISCUSSION THREAD

**Discussion by ExodistSKY1** [tsunamayo/Starship-EVO#4169] [Suggestions]
TLDR, will you make reactors that are touching other reactors share fuel?

A few different interpretations as to why.
-a bit of a pain and annoying to have to pipe fuel to every reactor, warp core or anything that will need some sort of fuel when they are all in one specific location

-Make it ideal for players to have their reactors/bricks that use fuel to be clumped and in one general area if they are trying to be efficient. 

-Possibly highly volatile when hit during combat ;) ;)

**Comment by tsunamayo** [comment]
Hi, it will not be possible as it, but I will provide a 0 height pipe for that purpose, with no fuel transfert limit.

**Reply by ExodistSKY1** [reply]
okay this will work! thank you
